HW1/P3/P3.txt

```diff
@@ -1 +1 @@
-# Your discussion here
+[((u'a', u'e', u'r', u's', u't'), 11, [u'arets', u'aster', u'rates', u'reast', u'resat', u'stare', u'stear', u'strae', u'tares', u'tears', u'teras'])]
```

Comment:
Nice, I keyed by word rather than separate chars but that's the only difference.